test/setup-test-env.ts

```diff
@@ -1,15 +1,26 @@
 import { installGlobals } from "@remix-run/node";
 import "@testing-library/jest-dom/extend-expect";
-import { expect, afterEach } from "vitest";
+import { afterEach, beforeAll, expect } from "vitest";
 import { cleanup } from "@testing-library/react";
 import matchers from "@testing-library/jest-dom/matchers";
+import { server } from "../mocks/server";
```

Comment:
Similar to above comment ... can we add an `export` to the server object defined `mocks/index.ts` to reuse that instead of creating a new `server.ts`

```ts
export const server = setupServer();
```